ArtemisKit/Sources/Messages/Services/MessagesService/MessagesService.swift

```diff
@@ -179,5 +179,5 @@ extension MessagesService {
 }
 
 enum MessagesServiceFactory {
-    static let shared: MessagesService = MessagesServiceImpl()
+    static let shared: MessagesService = CommandLine.arguments.contains("-Screenshots") ? MessagesServiceStub() : MessagesServiceImpl()
```

Comment:
I am wondering if we can contain this better and this library does it, but it's not a small dependency.

https://github.com/pointfreeco/swift-dependencies

Reply:
Good idea, I'll look into it. I wasn't too happy about this either since it's not easy to change, but something like a global variable also didn't sound too great. Maybe I can create a property wrapper or something to work, then we don't need a huge dependency. Will look into it :)